yolopandas/llm_accessor.py

```diff
@@ -31,7 +31,8 @@ def reset_chain(
     def query(self, query: str, yolo: bool = False) -> Any:
         """Query the dataframe."""
         df = self.df
-        inputs = {"query": query, "df_head": df.head(), "stop": "```"}
+        df_columns = df.columns
```

Comment:
What do you think about casting the columns to a list?
```suggestion
        df_columns = df.columns.tolist()
```
As is, the prompt reads
```
The dataframe has the following columns: Index(['column_1', 'column_2', ...], dtype='object').
```
If we cast to list, the prompt reads:
```
The dataframe has the following columns: ['column_1', 'column_2', ...].
```
which I think is slightly cleaner, although it's a minor thing. Tests appear to pass either way.

Reply:
@ccurme makes sense. Just needed to test it first but looks like there's no difference. Should be good now 👍🏽